src/lib/storage/fixed_string_dictionary_column.cpp

```diff
@@ -34,7 +34,7 @@ const AllTypeVariant FixedStringDictionaryColumn<T>::operator[](const ChunkOffse
     return NULL_VALUE;
   }
 
-  return _dictionary->get_string_at(value_id);
+  return AllTypeVariant{std::move(_dictionary->get_string_at(value_id))};
```

Comment:
That little change made such a big difference!

Reply:
Sometimes I wonder if C++ would have been a better language if you had to explicitly copy things.

https://pbs.twimg.com/media/DY5kURUU8AIizcd.jpg:large